data/formats-data.js

```diff
@@ -817,7 +817,7 @@ exports.BattleFormatsData = {
 	},
 	gengarmega: {
 		randomBattleMoves: ["shadowball","sludgewave","focusblast","taunt","destinybond","disable","perishsong","protect"],
-		randomDoubleBattleMoves: ["shadowball","sludgebomb","focusblast","thunderbolt","substitute","disable","taunt","hypnosis","gigadrain","trick","dazzlinggleam","protect"],
+		randomDoubleBattleMoves: ["shadowball","sludgebomb","focusblast","substitute","disable","taunt","hypnosis","willowisp","dazzlinggleam","protect"],
```

Comment:
Perish Song has only 8% usage on Gengar in Doubles. It's probably uncommon in Doubles because ganging up on Mega Gengar makes it harder to do perish trapping, since one Pokemon can break your sub and the other one can go for the kill, allowing them to switch out.